documentation/proc-pages/eng-models/heating_and_current_drive/RF/cutoff_ecrh.md

```diff
@@ -0,0 +1,63 @@
+# ECRH with Cutoff
+
+- `iefrf/iefrffix` = 13
+
+| Input | Description |
+|-------|-------------|
+| `dene`, $n_{\text{e}}$ | Avergae electron temperature $\left[10^{19}\text{m}^{-3}\right]$ |
+| `te`, $T_{\text{e}}$ | Avergae electron temperature $\left[\text{keV}\right]$ |
+| `rmajor`, $R_0$ | Major radius $\left[\text{m}\right]$ |
+| `bt`, $B_{\text{T}}$ | Toroidal magnetic field $\left[\text{T}\right]$ |
+| `zeff`, $Z_{\text{eff}}$ | Effective charge |
+| `harnum` | Harmonic number |
+| `mode` | RF mode |
+
+----
+
+
+
+$$
+\mathtt{fc} = \frac{1}{2\pi}eB_{\text{T}}m_{\text{e}}
```

Comment:
I think it should be `/m_e` at the end of this equation?

Reply:
Corrected